internal/gcsx/bucket_manager_test.go

```diff
@@ -56,41 +54,13 @@ func (t *BucketManagerTest) TestNewBucketManagerMethod() {
 		DebugGCS:                           true,
 		AppendThreshold:                    2,
 		TmpObjectPrefix:                    "TmpObjectPrefix",
-		EnableStorageClientLibrary:         true,
 	}
 
-	bm := NewBucketManager(bucketConfig, nil, t.storageHandle)
+	bm := NewBucketManager(bucketConfig, t.storageHandle)
 
 	ExpectNe(nil, bm)
 }
 
-func (t *BucketManagerTest) TestSetupGcsBucketWhenEnableStorageClientLibraryIsTrue() {
```

Comment:
shouldn't we keep this as there is no TestSetupGcsBucketMethod (i.e. which tests SetupGcsBucket) ?